flagsmith-es/react/index.js.map

```diff
@@ -1 +1 @@
EAAOT,SAAQ,SAAAW,GACXH,EAAKS,IAAIN,EAAYL,WAEtB,QAAyC,IAA9Bd,KAAKW,cAAcE,GACjC,QAAkB,IAAPC,EAAoB,CAC3B,IAAMO,EAAQrB,KAAKW,cAAcE,GAAOS,QAAQR,GAC5CO,GAAS,GACTrB,KAAKW,cAAcE,GAAOU,OAAOF,EAAO,QAG5CrB,KAAKW,cAAcE,GAAS,GAIxC,OAAOb,MAGXU,EAAAd,UAAA8B,QAAA,eAAA,IA+BCV,EAAAhB,KA/BcC,EAAA,GAAAC,EAAA,EAAPA,EAAOT,UAAAC,OAAPQ,IAAAD,EAAOC,GAAAT,UAAAS,GACX,GAAID,EAAKP,OAAS,EAAG,CACjB,IAAIiC,EACJ,GAAuB,iBAAZ1B,EAAK,GAAiB,CACvB,IAAA2B,EAAS3B,EAAK,QAChB4B,EAAgB5B,EAAK,GACrBA,EAAK,aAAc6B,QACnBF,EAAO3B,EAAK,GAAG,GACf4B,EAAgB5B,EAAK,GAAG,KAE5B0B,EAAQjB,EAAaqB,YAAYH,EAAM,CACnCI,SAAS,EACTC,YAAY,KAEVJ,cAAgBA,OAEtBF,EAAQjB,EAAaqB,YAAY9B,EAAK,GAAI,CACtC+B,SAAS,EACTC,YAAY,KAEVJ,cAAgB,UAEoB,IAAnC7B,KAAKW,cAAcgB,EAAMC,QAChC3B,EAAKiC,QACLjC,EAAKkC,QAAQR,GACb3B,KAAKW,cAAcgB,EAAMC,MAAMpB,SAAQ,SAAAM,GACnCA,EAAGf,MAAMiB,EAAMf,OAI3B,OAAOD,MAGJU,EAAAqB,YAAP,SAAsBH,EAAMQ,GAExB,QAFwB,IAAAA,IAAAA,EAA0B,IAClDA,EAAShD,EAAO,CAAE4C,SAAS,EAAOC,YAAY,EAAOI,YAAQC,GAAaF,GACpD,oBAAXG,OAAwB,CAC/B,GAAkC,mBAAvBA,OAAOR,YACd,OAAO,IAAIQ,OAAOR,YAAYH,EAAMQ,GAGxC,IAAMI,EAAc,SAACC,EAAOC,QAAA,IAAAA,IAAAA,EAA2B,IACnD,IAAM7B,EAAQ8B,SAASC,YAAY,eAEnC,OADA/B,EAAMgC,gBAAgBjB,EAAMc,EAAQV,QAASU,EAAQT,WAAYS,EAAQL,QAClExB,GAGX,OADA2B,EAAY5C,UAAY2C,OAAOO,MAAMlD,UAC9B4C,EAAYZ,EAAMQ,GAE7B,OAAOhD,EAAO,CAAEwC,KAAIA,EAAEC,cAAe,MAAQO,IAEpD1B,KChHKO,EAAS,IAAIP,EAINqC,EAAmBC,EAAiC,MAQpDC,EAA8C,SAACC,OAC3DC,EAASD,EAAAC,UAAEC,EAAOF,EAAAE,QAAEC,EAAWH,EAAAG,YAAEC,EAAQJ,EAAAI,SAEhCC,EAAiBC,GAAO,GAoB9B,OAnBIH,IAAgBF,EAAUM,aAC1BN,EAAUO,SAASL,GAEnBE,EAAeI,UACfJ,EAAeI,SAAU,EACrBP,EACAD,EAAUS,KAAI1E,EAAAA,EAAA,GACPkE,GAAO,CACVS,SAAU,eAAC,IAAO5D,EAAA,GAAAC,EAAA,EAAPA,EAAOT,UAAAC,OAAPQ,IAAAD,EAAOC,GAAAT,UAAAS,GACVkD,EAAQS,UACRT,EAAQS,SAAQ9D,MAAhBqD,EAAoBnD,GAExBgB,EAAOS,QAAQ,aAIvByB,EAAUzB,QAAU,WAAI,OAAAT,EAAOS,QAAQ,WAI3CoC,EAAAC,cAAChB,EAAiBiB,SAAQ,CAACC,MAAOd,GAC7BG,IAKPY,EAAc,SAAaD,GAC7B,IAAME,EAAMX,EAAOS,GAInB,OAHKE,EAAIR,UACLQ,EAAIR,QAAUM,GAEXE,EAAIR,SAITS,EAAe,SAACC,GAClB,GAAsB,iBAAXA,EACP,MAAO,CAACA,GACL,GAAsB,iBAAXA,GAEVA,EAAOxE,eAAe,UACtB,OAAOwE,EAGf,MAAM,IAAIC,MACN,6FAIFC,EAAe,SAACpB,EAAuBqB,EAAiBC,GAC1D,YAD0D,IAAAA,IAAAA,EAAqB,IACxED,EACFE,KAAI,SAACjE,GACF,MAAO,GAAGkE,OAAAxB,EAAUyB,SAASnE,IAAKkE,OAAAxB,EAAU0B,WAAWpE,OACxDkE,OAAOF,EAAOC,KAAI,SAACrF,GAAM,MACxB,GAAGsF,OAAAxB,EAAU2B,SAASzF,QAEzB0F,KAAK,MAGE,SAAAC,EAA6CX,EAAsBY,QAAA,IAAAA,IAAAA,EAA0B,IAKzG,IAAMT,EAAQN,EAAsBE,EAAaC,IAC3CI,EAASP,EAAsBE,EAAaa,IAC5C9B,EAAY+B,EAAWnC,GACvBG,EAA4BiC,EAC9BZ,EAAapB,EAAWqB,IADrBY,EAASlC,EAAA,GAAEmC,OAGZC,EAAY9B,EAAe4B,GAC3BG,EAAgBC,GAAY,WAC9B,IAAMC,EAAelB,EAAapB,EAAWqB,EAAOC,GAChDgB,IAAiBH,EAAU3B,UAC3B2B,EAAU3B,QAAU8B,EACpBJ,EAAaI,MAElB,IAoBH,OAnBAC,GAAU,WAEN,OADAzE,EAAOL,GAAG,QAAS2E,GACZ,WACHtE,EAAOQ,IAAI,QAAS8D,MAEzB,IACSI,GAAQ,WAChB,IAAMC,EAAW,GASjB,OARApB,EAAME,KAAI,SAACjE,GACPmF,EAAInF,GAAK,CACLoF,QAAS1C,EAAW0B,WAAWpE,GAC/BwD,MAAOd,EAAWyB,SAASnE,OAEhCkE,OAAOF,MAAAA,OAAM,EAANA,EAAQC,KAAI,SAACoB,GACnBF,EAAIE,GAAK3C,EAAU2B,SAASgB,OAEzBF,IACR,CAACR,IAKK,IAAAW,EAAe,WACxB,IAAMC,EAAUd,EAAuBnC,GAEvC,IAAKiD,EACD,MAAM,IAAI1B,MAAM,yDAGpB,OAAO0B"}
```

Comment:
I’m not really used to seeing sourcemaps or the transpiled JS checked in. In my libs I have them sent to a dist folder that is git ignored. This helps to reduce potential drifts caused by someone modifying the TS but not running the build. I trust that your team makes sure it’s done, but it’s not conventional. I wouldn’t consider this a deal breaker for our company’s adoption. But I’m just sharing. Thanks again for removing the ts-ignores.